test/features/schema_metadata.feature

```diff
@@ -33,3 +33,63 @@ Feature: SchemaMetadata
         When I log in
         When I go to "/dataset/warandpeace"
         Then I should see an element with xpath "//link[@type='application/ld+json']"
+
+    Scenario Outline: Scenario 1: A publisher publishes a dataset [DQL3-1]
```

Comment:
Update all the scenario test titles to match the steps the test is doing.
We do not need use the same title as the Scenario listed as the tests in the story as they are too broad and don't always match the test scenario.
We don't need to reference the story number either

Reply:
refactored